61.md

```diff
@@ -0,0 +1,36 @@
+NIP-61
+======
+
+Danmaku
+-------
+
+`draft` `optional`
+
+This NIP defines a new kind of event, `2333`, for danmaku.
+
+Danmaku (弾幕 だんまく; 弹幕; literally translated as "bullet curtain" and figuratively as "barrage") is a subtitle system in online video platforms that originates from Japan and popularised in Mainland China. Such system allows user to post moving comments onto a playing video that are synchronized to the video timeline. The comments are typically presented as “shooting” across the screen, resembling a barrage.
```

Comment:
Background can be elided, how comments are displayed should be up to the client (I would find this very useful, but would _not_ want the comments to be flying past).

Reply:
Sure. I'm worried that some people might not know what "danmaku" refers to, so I added some background information. Indeed, not all danmaku are flying past, lol. This should be handled by the client.

Reply:
Maybe a quick definition would be useful

Reply:
Alright, I'll add a brief definition.